schoolopy/main.py

```diff
@@ -1925,8 +1927,16 @@ def get_roles(self):
     def get_role(self, role_id):
         return Role(self._get('roles/%s' % role_id))
 
-    def get_messages(self):
-        return [MessageThread(raw) for raw in self._get('messages')['message']]
+    def get_messages(self, message_folder):
+        """
+        List messages in one of the valid message folders.
+
+        :param message_folder: Either 'sent' or 'inbox'
```

Comment:
I almost feel like this is worth creating an enum... not quite though.

Reply:
Originally I had a type annotation that only allowed 'sent' or 'inbox'. But I decided that having just one type annotation in the whole file would be weird.